scene/3d/audio_stream_player_3d.cpp

```diff
@@ -126,6 +126,7 @@ void AudioStreamPlayer3D::_mix_audio() {
 			current.filter.set_gain(current.filter_gain);
 
 			if (interpolate_filter) {
+				ERR_FAIL_COND(k * 2 >= 6);
```

Comment:
Where does the magic number come from? It feels like if this condition can be triggered, it should have been validated further up (e.g. when getting `buffers`).

Reply:
Not related to this change, but there's a stray semicolon on line 96 that would be worth removing :)

Reply:
I'm not sure how this exactly should look like, but probably best idea is to extend filter_process array to 8 items, which will covers all 4 channel instead add an overflow warning because filter_process is used only in this one function.

In audio_effect_filter.cpp is also filter_process array which have 8 elements([2][4] array)